install_scripts/instrument_upgrade.bat

```diff
@@ -0,0 +1,14 @@
+REM Upgrade NDXDEMO for sprint review
+
+set "SERVER_SOURCE=\\isis.cclrc.ac.uk\inst$\Kits$\CompGroup\ICP\EPICS\EPICS_win7_x64\BUILD-3125\EPICS"
+set "CLIENT_SOURCE=\\isis.cclrc.ac.uk\inst$\Kits$\CompGroup\ICP\Client\BUILD179"
+set "GENIE_PYTHON=\\isis.cclrc.ac.uk\inst$\Kits$\CompGroup\ICP\genie_python\BUILD-146"
+
+set "STOP_IBEX=C:\Instrument\Apps\EPICS\stop_ibex_server"
+set "START_IBEX=C:\Instrument\Apps\EPICS\start_ibex_server"
+REM IF EXIST "C:\Instrument\Apps\EPICS" (start /wait cmd /c "%STOP_IBEX%")
+
+call "%GENIE_PYTHON%\Python\python.exe" IBEX_upgrade.py --server_dir "%SERVER_SOURCE%" --client_dir "%CLIENT_SOURCE%" --confirm_step instrument_update
```

Comment:
`instrument_update` does not install anything, yet we have to provide these dirs. Just spoke to John and he doesn't want to rewrite the underlying script to not require the arguments since that's the intended destination. Please could you add a warning on `instrument_update` that the user must acknowledge specifying that nothing will be installed. That will avoid an unfortunate situation down the line where we expect things to be installed that aren't